frontend/src/pages/HomePage/homePage.jsx

```diff
@@ -0,0 +1,15 @@
+import NavBar from '../../components/HomePage/NavBar/navBar';
+import HomePageBody from '../../components/HomePage/Body/homePageBody';
+import UserProfile from '../../components/HomePage/Profiles/profiles';
+
+function Home() {
+    return (
+        <div>
+            <NavBar />
+            <HomePageBody />
+            <UserProfile />
```

Comment:
now that this component require properties, you should pass the user data to the user prop. you can pass some dummy data like:

```js
const dummyUser = { firstName: "john", lastName: "doe" } 
<UserProfile user={dummyUser}/>

```